pkg/kubelet/kubelet.go

```diff
@@ -2138,9 +2150,15 @@ func (kl *Kubelet) syncLoopIteration(updates <-chan kubetypes.PodUpdate, handler
 			glog.Errorf("Kubelet does not support snapshot update")
 		}
 	case <-kl.resyncTicker.C:
```

Comment:
I think kubelet.go is too large and holds too much state currently, so I'm generally in favor of any ways to move state (the ticker, in this case) out of kubelet. That said, I recognize it does add non-trivial complexity to the WorkQueue, so whichever way you want to go is fine with me.